_docs/ops/security-ir.md

```diff
@@ -61,19 +77,38 @@ For full details, read on.
 
 An incident begins when someone becomes aware of a potential incident. We define "incident" broadly, following [NIST SP 800-61](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-61r2.pdf), as "a violation or imminent threat of violation of computer security policies, acceptable use policies, or standard security practices" (6). This is a deliberately broad definition, designed to encompass any scenario that might threaten the security of cloud.gov.
 
-When a person outside the cloud.gov team (the *reporter*) notices a cloud.gov-related incident, they should begin reporting it by using the [TTS incident response process](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/security-incidents/), and then post about it in [`#cloud-gov`](https://gsa-tts.slack.com/messages/cloud-gov/) using `@cg-team`. If they don't get acknowledgment from the cloud.gov team right away, they should escalate by contacting the cloud.gov leads directly until they receive acknowledgment of their report.
+When a TTS staff member (the *reporter*) notices and reports a cloud.gov-related incident using the [TTS incident response process](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/security-incidents/) they should then  notify the cloud.gov team in [`#cg-support`](https://gsa-tts.slack.com/archives/C09CR1Q9Z)
+
+When a cloud.gov team member is the first person to notice an incident, they should 
+join [`#cg-incidents`](https://gsa-tts.slack.com/archives/GTNBK2L9K), and use the "Declare Incident" Workflow.
 
-When a cloud.gov team member is the first person to notice an incident, they should also begin reporting it by using the [TTS incident response process](https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/security-incidents/) and posting about it in [`#cloud-gov`](https://gsa-tts.slack.com/messages/cloud-gov/) using `@cg-team` (including notifying the cloud.gov leads).
+Declaring an incident results in further guidance to:
+  - Designate an Incident Commander and Scribe
+  - Claim an incident folder in our "Incident Response" [Google Drive folder](https://drive.google.com/drive/folders/1WtLFiZuxLmKR4mrztEE9YtS78nGxC--P)
+    - We try to populate incident folders in advance based on existing templates
+  - Share the URL of the incident document
+  - Update incident doc in that folder with comms and key findings.
+  - Gather in a Google Meet
+  - Set reminders to send comms updates, and to rotate key roles every 6 hours.
```

Comment:
This section feels broadly repetitive of https://github.com/cloud-gov/cg-site/pull/2429/files#diff-8b21e674168ee96c2cb0966300d0c51b63e5fd2389228d91f3d3cf45df23d510R39-R48

Is there any way to avoid that?

Reply:
The IR document is still very duplicative. I’d like to do another round and leverage somthing like https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2 to expand/collapse the details.

Reply:
Fair. As a first pass, this is a big improvement